test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExe.cs

```diff
@@ -68,19 +68,18 @@ public void It_builds_with_inferred_platform_target(string runtimeIdentifier, st
                 .WithSource()
                 .Restore("", $"/p:RuntimeIdentifier={runtimeIdentifier}");
 
-            var buildCommand = new BuildCommand(Stage0MSBuild, testAsset.TestRoot);
-            buildCommand
+            var getValuesCommand = new GetValuesCommand(Stage0MSBuild, testAsset.TestRoot,
+                "net46", "PlatformTarget", GetValuesCommand.ValueType.Property);
+
+            getValuesCommand
                 .Execute($"/p:RuntimeIdentifier={runtimeIdentifier}")
                 .Should()
                 .Pass();
 
-            var assemblyPath = Path.Combine(buildCommand.GetOutputDirectory("net46").FullName, "DesktopMinusRid.exe");
-            var assemblyInfo = AssemblyInfo.Get(assemblyPath);
-            assemblyInfo
+            getValuesCommand
+                .GetValues()
                 .Should()
-                .Contain(
-                    "AssemblyDescriptionAttribute",
-                    $"PlatformTarget={expectedPlatformTarget}");
+                .BeEquivalentTo(expectedPlatformTarget);
```

Comment:
I had to look it up, but it means the elements of the collection [can be in any order](https://github.com/dennisdoomen/FluentAssertions/blob/4f396c423b6084d2c9c01f09a11aed0b2f55309f/Src/Core/Collections/StringCollectionAssertions.cs#L35).